src/components/Heading/Heading.mdx

```diff
@@ -53,6 +54,20 @@ proper order and not skipping any. For example, the following is invalid HTML:
   language="html"
 />
 
+When either the overline or subtitle elements are rendered as part of the
+heading, then the whole lockup should be wrapped in an `<hgroup>` element with
```

Comment:
perhaps "will" is a better word than "should"? That is being done in the component.